spec/draft/profile-spec.md

```diff
@@ -340,6 +340,85 @@ model User {
 }
 ```
 
+# Example
+
+Example : example Identifier { ExampleInput ExampleOutput }
+
+ExampleInput : input ComlinkLiteral
+
+ExampleOutput : 
+- ExampleResult
+- ExampleError
+
+ExampleResult : result ComlinkLiteral
+
+ExampleError : error ComlinkLiteral
+
+# Literal
+
+## Comlink Literal
+
+ComlinkLiteral :
+- PrimitiveLiteral
+- ObjectLiteral
+- ArrayLiteral
+
+## Primitive Literal
+
+PrimitiveLiteral : StringValue | NumberLiteral | BooleanLiteral | NullLiteral
+
+NumberLiteral: NumberSign? NumberLiteralDigits
+
+NumberSign : `-` | `+`
+
+NumberLiteralDigits : NumberLiteralInteger | NumberLiteralFloat
+
+NumberLiteralInteger : NumberLiteralIntegerBaseTen | NumberLiteralIntegerBaseTwo | NumberLiteralIntegerBaseEight | NumberLiteralIntegerBaseSixteen
+
+NumberLiteralIntegerBaseTen : DigitBaseTen+
+
+NumberLiteralIntegerBaseTwo : 0b DigitBaseTwo+
+
+NumberLiteralIntegerBaseEight : 0o DigitBaseEight+
+
+NumberLiteralIntegerBaseSixteen : 0x DigitBaseSixteen+
+
+DigitBaseTen : /[0-9]/
+
+DigitBaseTwo : /[0-1]/
+
+DigitBaseEight : /[0-7]/
+
+DigitBaseSixteen : /[0-9a-fA-F]/
+
+NumberLiteralFloat : DigitBaseTen+ . DigitBaseTen+
+
+BooleanLiteral : `true` | `false`
+
+NullLiteral : `null`
+
+## Object Literal
+
+ObjectLiteral : { KeyValueAssignment+ }
```

Comment:
I think we might want to support empty object literals, so `{ KeyValueAssignment* }`?

Reply:
I changed this.